File: plugins/scandit-sdk/evals/sparkscan-net-android/fixtures/EmptyActivity.cs
using Android.OS;
using AndroidX.AppCompat.App;

namespace MyApp;

[Activity(MainLauncher = true, Label = "@string/app_name")]
public class MainActivity : AppCompatActivity
{
    protected override void OnCreate(Bundle? savedInstanceState)
    {
        base.OnCreate(savedInstanceState);
        this.SetContentView(Resource.Layout.activity_main);
    }
}

File: plugins/scandit-sdk/evals/matrixscan-count-android/fixtures/EmptyActivity.kt
package com.example.myapp

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

/**
 * A plain Android screen with no scanning yet. The user wants to add MatrixScan Count
 * (bulk barcode counting) here.
 */
class ScanActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val label = TextView(this).apply {
            text = "Inventory"
        }
        setContentView(label)
    }
}
